.github/workflows/execution-plan.yml

```diff
@@ -0,0 +1,30 @@
+name: "Execution Plan"
+'on':
+  pull_request:
+    branches:
+      - main
+jobs:
+  execution-plan:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+      - uses: google-github-actions/auth@v0 
+        name: 'Authenticate to Google Cloud'
+        with:
+          credentials_json: ${{ secrets.FDC_CORE_CI_IMAGE_READER }} 
+      - name: gcloud authorize
+        run: |
+          gcloud auth configure-docker
+      - name: Get changed files
+        id: changes
+        run: |
+          FILES_CHANGED=$(git diff --name-only --diff-filter=ACMRT ${{ github.event.pull_request.base.sha }} ${{ github.sha }})
+          echo 'FILES_CHANGED<<EOF' >> $GITHUB_ENV
+          echo "$FILES_CHANGED" >> $GITHUB_ENV
+          echo 'EOF' >> $GITHUB_ENV
+      - name: Run Execution plan
+        run: make plan
```

Comment:
What was the intention, i.e. the goal? If it was to have as little as possible code in GitHub Actions or Jenkins specific files, I would propose to put it into a single script that computes the changed files and from it the execution plan. Since `${{ github.event.pull_request.base.sha }}` and `${{ github.sha }}` are GitHub Actions specific, you would have to pass that in as arguments. Passing them as environment variables `$GIT_BASE_COMMIT` and `$GIT_CURRENT_COMMIT` (e.g.) would be one option, but `$1` and `$2` seem more obvious to me, so I would opt for that.